vms/platformvm/state/staker.go

```diff
@@ -83,7 +83,7 @@ func (s *Staker) Less(than *Staker) bool {
 	return bytes.Compare(s.TxID[:], than.TxID[:]) == -1
 }
 
-func NewCurrentStaker(txID ids.ID, staker txs.Staker, potentialReward uint64) (*Staker, error) {
+func NewCurrentStaker(txID ids.ID, staker txs.ScheduledStaker, potentialReward uint64) (*Staker, error) {
```

Comment:
in next PR we'll change the signature of this function to use txs.Staker instead of txs.ScheduledStaker.